helper.py

```diff
@@ -141,21 +147,20 @@ def create_from_template(destination, board, libs):
     # Essentially need to:
     # * run copy_project on template and then rewrite the cpp file..
 
-    libs = pathlib.Path(libs).absolute().as_posix()
-    file_path = pathlib.Path(__file__).absolute().as_posix().replace('helper.py', '')
-    print("print", file_path)
+    libs = pathlib.Path(os.path.relpath(libs, destination)).as_posix()
+    print ('DESTINATEIONAAAA   ', libs, destination)
```

Comment:
what the heck is "DESTINATEIONAAAA"?

Reply:
That's a string that stands out really well when you're testing something haha. 😆